mkdocs.yml

```diff
@@ -240,7 +240,8 @@ extra:
         # Build-related string tokens
         kafkaversion: 2.7
         ksqldbversion: 0.16.0
-        kstreamsbetatag: 6.2.0-beta210122194818
+        kstreamsbetatag: 6.2.0-beta201122193350-cp5
```

Comment:
Hey @JimGalasyn , did you mean to change this tag on master? Not sure whether the old value was already prepped for 0.16 or not. If so, this change has just overridden it with the value for 0.15.

Also, `kstreamsbetabuild` below should be 1 in most cases (0.15 was a bit special).